site/docs/encoding/scale.md

```diff
@@ -321,6 +321,38 @@ For example, the following plot has a binned field on the `color` channel.
+The domain is considered continuous and thus the scale will accept any reasonable input value; however, the domain is specified as a discrete set of sample values.
+The number of values in (i.e., the cardinality of) the output range determines the number of quantiles that will be computed from the domain. To compute the quantiles, the domain is sorted, and treated as a population of discrete values.
+The resulting quantile boundaries segment the domain into groups with roughly equals numbers of sample points per group.
+
+Quantile scales are particularly useful for creating color or size encodings with a fixed number of output values. Using a discrete set of encoding levels (typically between 5-9 colors or sizes) sometimes supports more accurate perceptual comparison than a continuous range.
+For related functionality see [quantize scales](https://vega.github.io/vega-lite/docs/scale.html#quantize), which partition the domain into uniform domain extents, rather than groups with equal element counts.
+Quantile scales have the benefit of evenly distributing data points to encoded values. In contrast, quantize scales uniformly segment the input domain and provide no guarantee on how data points will be distributed among the output visual values.
+
+<span class="vl-example" data-name="bar_scale_quantile"></span>
+
+{:#quantize}
+### Quantize Scales
+
+Quantize scales (`"quantize"`) are similar to [linear scales](https://vega.github.io/vega-lite/docs/scale.html#linear), except they use a discrete rather than continuous range. The continuous input domain is divided into uniform segments based on the number of values in (i.e., the cardinality of) the output range.
+Each range value y can be expressed as a quantized linear function of the domain value x: _y_ = _m round(x) + b_.
+
+Quantize scales are particularly useful for creating color or size encodings with a fixed number of output values. Using a discrete set of encoding levels (typically between 5-9 colors or sizes) sometimes supports more accurate perceptual comparison than a continuous range.
+For related functionality see [quantile scales](https://vega.github.io/vega-lite/docs/scale.html#quantile), which partition the domain into groups with equal element counts, rather than uniform domain extents.
+
+<span class="vl-example" data-name="bar_scale_quantize"></span>
+
+{:#threshold}
+### Threshold Scales
+
+Threshold scales (`"threshold"`) are similar to [quantize scales](https://vega.github.io/vega-lite/docs/scale.html#quantize), except they allow mapping of arbitrary subsets of the domain (not uniform segments) to discrete values in the range.
+The input domain is still continuous, and divided into slices based on a set of threshold values provided to the domain property. The range property must have N+1 elements, where N is the number of threshold boundaries provided in the domain.
```

Comment:
"The range property, if specified, ..."

Perhaps, it's worth discussing the default range for all of these scales?

Reply:
I think the default range should be a uniform threshold boundaries with certain number, e.g. 4

Reply:
I mean discuss / "describe" on the docs so readers understand the scale's default behavior. :)